src/lib.rs

```diff
@@ -6,6 +6,7 @@
 
 //! BlindBid impl
 #![allow(non_snake_case)]
+#![deny(warnings)]
```

Comment:
`deny(warnings)` should be part of the CI and not a restriction set to the development environment

Reply:
Why? It really helps to develop. Otherways:
- Either we get PR's that pass with warnings.
- You push code and in some feature selection that you haven't tested a warning appears.

Reply:
When you are developing and running constant `cargo check` commands, these will fail if you have unfinished code.

The rule of thumb is to not push anything with warnings, that's why the responsibility to deny warnings is closer to the CI compared to the development time.